src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2,10 +2,11 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics;
+#if FEATURE_CULTUREINFO_CONSOLE_FALLBACK
 using System.Globalization;
+#endif
```

Comment:
Looks like this is now available everywhere too, so we can delete the feature flag.